### PR TITLE
[nudge-a-palooza] Enable dark themes upsell nudges for everyone

### PR DESCRIPTION
**Pull request by adamziel**
Related to p9jf6J-Hl-p2

Updated nudges in `test` variation of themesNudgesUpdates won. This PR merges updated upsell nudges. There is less certainty about a nudge in the checkout process so we are moving it to it's own separate test.

Test plan:
1. As a free user, go to /themes/:site and confirm the premium upsell nudge is dark
1. Go to /themes/:theme/:site for a paid theme and confirm the premium upsell nudge is there and is dark
1. Click on theme preview and confirm the upsell nudge is there

### CODE REVIEW

File: client/my-sites/theme/main.jsx [hunk 1]
```diff
@@ -638,25 +637,18 @@ class ThemeSheet extends React.Component {
...
-					title={ 'Access this theme for FREE with a Premium or Business plan!' }
-					description={
+					title={ translate( 'Access this theme for FREE with a Premium or Business plan!' ) }
+					description={ translate(
 						'Instantly unlock all premium themes, more storage space, advanced customization, video support, and more when you upgrade.'
-					}
+					) }
 					event="themes_plan_particular_free_with_plan"
-					callToAction={ 'View plans' }
+					callToAction={ translate( 'View plans' ) }
```
Comment (a8ci18n) [inline client/my-sites/theme/main.jsx]: Hi! I've found a possible matching string that has already been translated **34** times:
`translate( 'View Plans' )` *ES Score: 13*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).

**Comment by dzver** [comment]
I see a JavaScript error on http://calypso.localhost:3000/theme/photo-blog/:site

**Comment by adamziel** [comment]
@dzver Updated and fixed

File: client/components/mini-site-preview/index.jsx [hunk 2]
```diff
@@ -6,7 +6,7 @@ import React from 'react';
 import PropTypes from 'prop-types';
 import classnames from 'classnames';
 
-const MiniSitePreview = ( { className, imageSrc  } ) =>
+const MiniSitePreview = ( { className, imageSrc } ) =>
```
Comment (adamziel) [inline client/components/mini-site-preview/index.jsx]: ESLint did that on merge :-( it has to stay now

**Pull request merged**